packages/openapi-generator/src/options/options.ts

```diff
@@ -37,5 +42,11 @@ export const cliOptions = {
       'When enabled, all generated files follow the ECMAScript module syntax.',
     type: 'boolean',
     default: false
+  },
+  schemaPrefix: {
+    describe:
+      'When enabled, all generated files follow the ECMAScript module syntax.',
+    type: 'string',
+    default: undefined
```

Comment:
[q] Would it be easier to handle if it was `''`? Then you wouldn't need any special handling when using the prefix.